cmd/gokr-packer/packer.go

```diff
@@ -281,6 +303,130 @@ func derivePartUUID(hostname string) uint32 {
 	return h.Sum32()
 }
 
+func generateAndSignCert() ([]byte, *rsa.PrivateKey, error) {
```

Comment:
All the certificate related functions should probably go into their own certs.go file. This file is already pretty long.